kubernetes-addons/prometheus/values.yaml

```diff
@@ -1,15 +1,57 @@
 serviceAccounts:
```

Comment:
Looks like you removed logic above to leverage this values file...?

Reply:
good spot. let me add that fix